README.md

```diff
@@ -1 +1,170 @@
 # Attestation Evidence manipulation tool
+
+## Installing and configuring
+
+To install the `evcli` command, do:
+
+```shell
+go install github.com/veraison/evcli@latest
+```
+
+To configure auto-completion, use the `completion` subcommand.  For example, if `bash` is your shell, you would do something like:
+
+```shell
+evcli completion bash > ~/.bash_completion.d/evcli
+. ~/.bash_completion
+```
+
+If instead you are using `zsh` managed via [ohmyzsh](https://ohmyz.sh):
+
+```shell
+evcli completion zsh > ~/.oh-my-zsh/completions/_evcli
+. ~/.zshrc
+```
+
+For more help on completion:
+
+```shell
+evcli completion --help
+```
+
+## PSA attestation tokens manipulation
+
+The `psa` subcommand allows you to [create](#create), [check](#check) and [verify](#verify) PSA attestation tokens.
+
+### Create
+
+Use the `psa create` subcommand to create a PSA attestation token from the supplied claims in JSON format and IAK in JSON Web Key (JWK) format<sup>[1](#inputs-ex)</sup>.
+
+```shell
+evcli psa create \
+    --claims=psa-claims-profile-2.json \
+    --key=ec256.json
+```
+
+On success, you should see the following printed to stdout:
+
+```
+>> "psa-claims-profile-2.cbor" successfully created
+```
+
+The CBOR-encoded PSA token is stored in the current working directory with a name derived from the claims file you supplied.  If you want, you can specify a different name using the `--token` command line switch (abbrev. `-t`).
+
+For example:
+
+```shell
+evcli psa create \
+    --claims=psa-claims-profile-2.json \
+    --key=ec256.json \
+    --token=my.cbor
+```
+
+By default, PSA tokens are created according to the "http://arm.com/psa/2.0.0" profile.  If you are using the legacy "PSA_IOT_PROFILE_1" instead, you will need to explicitly pass it via the command line using the `--profile` switch (abbrev.  `-p`):
+
+```shell
+evcli psa create \
+    --claims=psa-claims-profile-1.json \
+    --key=ec256.json \
+    --profile=PSA_IOT_PROFILE_1
+```
+
+### Check
+
+Use the `psa check` subcommand to verify the cryptographic signature over the supplied PSA attestation token as well as checking whether its claim set is well-formed.
+
+To check the PSA attestation token in my.cbor using the public key in es256.json:
+
+
+```shell
+evcli psa check \
+    --token=my.cbor \
+    --key=es256.json
+```
+
+A message will indicate whether the signature has been successfully verified:
+
+```console
+>> "my.cbor" verified
+```
+
+In such case, the claim set is printed to stdout in JSON format:
+
+```json
+>> embedded claims:
```

Comment:
Seems line 93 does not come up nicely in the redered .md also, needs correction on this one!

Reply:
> Seems line 93 does not come up nicely in the redered .md

yes, JSON highlighting is not very tolerant :-)

> also, needs correction on this one!

I don't understand what needs correcting?